library/fixed_point/qsharp.json

```diff
@@ -0,0 +1,35 @@
+{
+  "author": "Microsoft",
+  "license": "MIT",
+  "dependencies": {
+    "Signed": {
+      "github": {
+        "owner": "microsoft",
+        "repo": "qsharp",
+        "ref": "f3b331ff",
+        "path": "library/signed"
+      }
+    },
+    "Unstable": {
+      "github": {
+        "owner": "Microsoft",
+        "repo": "qsharp",
+        "ref": "d5f0fbcf",
+        "path": "library/unstable"
+      }
+    }
```

Comment:
nitpick: It's not a problem, but does read a little odd that the two libraries this depends on come from the same repo but with different refs.

Reply:
It was because #1888 wasn't in yet. I'll update it now that that's merged.

Reply:
addressed in [3d6bbcd](https://github.com/microsoft/qsharp/pull/1838/commits/3d6bbcd9c577fbb0b81431fa2f945de0eede01fd)